src/ProfileManager.ts

```diff
@@ -242,7 +242,7 @@ export class ProfileManager implements IProfileManager {
           if (amtProfile.ciraConfigObject?.password) { amtProfile.ciraConfigObject.password = await this.configurator.secretsManager.getSecretFromKey(`${EnvReader.GlobalEnvConfig.VaultConfig.SecretsPath}CIRAConfigs/${amtProfile.ciraConfigObject.configName}`, amtProfile.ciraConfigObject.password) } else { this.logger.error("The amtProfile CIRAConfigObject doesn't have a password. Check CIRA profile creation.") }
```

Comment:
So looking at the code analysis tool --- what is in the `amtProfile.ciraConfigObject.password` field if not the password?

Reply:
yes it is the password. From my understanding, it was flagged because the password was being logged. I removed all the object logging in ProfileManager and only log the name of the profile.

Reply:
ok, so it looks like this still needs to be fixed. `amtProfile.ciraConfigObject.password` shouldn't be the key in vault. Additionally, since getSecretFromKey logs whats passed in, the password is getting logged.

Reply:
i removed the key logging in the SecretManagerService.